cdragontoolbox/binfile.py

```diff
@@ -361,6 +402,33 @@ def read_binfile_entry(self, htype):
         assert self.f.tell() - pos == length
         return entry
 
+    def read_patch_section(self):
+        """Reads the entire PTCH section"""
+
+        count = self.read_u32()
+        patch_entries = {}
+        for _ in range(count):
+            hpath = self.read_fmt('<2L')[0]
+            btype = self.parse_bintype(self.read_u8())
+            string = self.read_string()
```

Comment:
Not sure how to describe it, it's like a nested class name, e.g. `mAtlas.mTextureSourceResolutionHeight`.
Probably something like `objectpath`?

It would also be possible to not interpret a `.` as hierarchy change, but I believe that makes sense, especially seeing as how other structs with the name `mAtlas` for example exist.

Reply:
`objectpath` seems nice. 👍